Automattic/wp-calypso
# Update Jetpack plans lightbox

Related to 1201210512993648-as-1204545909511089

## Proposed Changes

Figma 0MMYGZ7bRaHuwMF3fFS2VC-fi-3402_32633

* Add the "FAQ" and "Also Included" sections to the bundle lightbox
* Add callouts to each "Included" product in the bundle lightbox
* Fix FoldableCard height on mobile

|  | Starter | Security | Complete |
|:---:|:---:|:---:|:---:|
| Desktop | <img width="1098" alt="image" src="https://github.com/Automattic/wp-calypso/assets/5714212/b562dd4a-1dd6-4c78-9bd8-dcfab2e48edf"> | <img width="1093" alt="image" src="https://github.com/Automattic/wp-calypso/assets/5714212/ef79e91a-7804-4414-9119-626fbb17a7b4"> |<img width="1092" alt="image" src="https://github.com/Automattic/wp-calypso/assets/5714212/2d602831-404a-42bf-b3ac-07f8961a6234"> |
| Mobile | <img width="324" alt="image" src="https://github.com/Automattic/wp-calypso/assets/5714212/1f8c8720-5c8b-4779-9fd5-af870e55f483"> | <img width="369" alt="image" src="https://github.com/Automattic/wp-calypso/assets/5714212/7ea63c5b-aa69-4d60-b4b5-3b55e0c4a732"> | <img width="372" alt="image" src="https://github.com/Automattic/wp-calypso/assets/5714212/c2262c77-d107-443c-9836-1c5e04795deb"> |


## Testing Instructions



* Visit the live link for Jetpack Cloud
* Visit /pricing and /features/comparison
* Click on the "More about {bundle name}"
* Make sure the lightbox works and its design matches the screenshots above and the proposed design in the Figma file

## Pre-merge Checklist
